src/changefile/writeChangeFiles.ts

```diff
@@ -3,7 +3,7 @@ import { getChangePath } from '../paths';
 import { getBranchName, stage, commit } from 'workspace-tools';
 import fs from 'fs-extra';
 import path from 'path';
-import { v4 as uuidv4 } from 'uuid';
+import humanId from 'human-id';
```

Comment:
Would it be possible to add a repo option `humanId?: boolean` and only use `human-id` if that's enabled? I don't think it's desirable to name change files with random words in all cases (I wouldn't want this), and regardless, suddenly changing to that might catch people off guard.

Reply:
I got this idea from https://github.com/atlassian/changesets/blob/eae72367e9cb8e708a1d441770dbc70930d4e1ad/packages/write/src/index.ts#L17 and personally find it much more readable than a guid, but that's up to you and @kenotron.

Reply:
I think this is completely subjective. The ONLY reason why I wouldn't do it is the addition of yet another dependency. Maybe we should just revert it for now and add it as a neat nice-to-have feature?

Reply:
I'll create a separate PR to talk about that. Don't wanna block this one on this.

Reply:
> I think this is completely subjective. The ONLY reason why I wouldn't do it is the addition of yet another dependency. Maybe we should just revert it for now and add it as a neat nice-to-have feature?

Definitely agree that it's subjective (and I'm not concerned about the extra dep). It just seems like an especially polarizing type of subjective, as demonstrated by both of you liking it and me thinking "eew this is weird."